### PR TITLE
Update JS dependencies

### PR DESCRIPTION
**Pull request by silverwind**
A bugfix for JS syntax in highlight.js and some misc updates.

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -50,12 +50,12 @@
     "worker-loader": "2.0.0"
   },
   "devDependencies": {
-    "eslint": "6.8.0",
+    "eslint": "7.0.0",
```
Comment (CirnoT) [inline package.json]: Not convinced that `eslint-config-airbnb-base` support ESLint v7, some changes in their rules might be necessary; tracked on https://github.com/airbnb/javascript/issues/2223

`eslint-plugin-import` looks fine aside from single deprecation https://github.com/benmosher/eslint-plugin-import/commit/c629186be2a4ac62303803c0214e699a83ee024f#diff-48c028a277414962bda6cdb19351d72bR45

Personally I'd wait on it.

Comment (silverwind) [reply package.json]: I don't see anything concerning there, and linting does work in 7.0. I guess their changes would mostly just be rule updates.

Another reason I want 7.0 is that I want to add [eslint-plugin-unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn) which requires it.

Comment (CirnoT) [reply package.json]: Some rules might be temporarily broken until they update it, I suppose if we are fine with it then go ahead.

Comment (silverwind) [reply package.json]: I think it's fine, the only potentially breaking issue I see is [this](https://eslint.org/docs/user-guide/migrating-to-7.0.0#rule-tester-strict) but given that we don't have new failures, we should be unaffected. I plan to update the plugins as soon as they release new versions.

**Comment by silverwind** [comment]
Undid `eslint` upgrade (waiting for plugins) and updated `swagger-ui` again.

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**